main.go

```diff
@@ -158,7 +158,7 @@ func main() {
 			log.Fatalf("Failed to start openshift sdn in node mode: %v", err)
 		}
 	} else if opts.master {
-		err := be.StartMaster(opts.sync, opts.containerNetwork, opts.containerSubnetLength)
+		err := be.StartMaster(opts.sync, opts.containerNetwork, opts.containerSubnetLength, "172.30.0.0/16")
```

Comment:
Instead of hard coding to "172.30.0.0/16", can we make this as config option and use "172.30.0.0/16" as default value if not explicitly specified? (similar to containerNetwork)